MAUI/Carousel-View/Transformation.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout : post
 title: Transformation in .NET MAUI Carousel View control | Syncfusion
-description: Learn here all about Transformation support in Syncfusion .NET MAUI Carousel View (SfCarousel) control and more.
+description: Learn here all about transformation support in Syncfusion .NET MAUI Carousel View (SfCarousel) control, its elements and more.
```

Comment:
revert this 
<img width="659" alt="image" src="https://github.com/syncfusion-content/maui-docs/assets/61148953/997fc96d-8eab-4cc3-933a-f082d5ae4672">

Reply:
changed